pilot/pkg/security/authz/model/model.go

```diff
@@ -157,7 +157,11 @@ func NewModelV1alpha1(trustDomainBundle trustdomain.Bundle, role *istio_rbac.Ser
 
 			property := KeyValues{}
 			for k, v := range subject.Properties {
-				property[k] = []string{v}
+				values := []string{v}
```

Comment:
This shouldn't be done in `model.go`, can you move it to `principal.go`?

Reply:
Oh, I didn't realize it's already added in other places, it's okay for this PR but could you put a TODO to refactor this?

Reply:
To be more clear, the refactor is to add a `func (principal *Principal) ReplaceTrustDomainAliases() that searches and replace all trust domains in the principal. Then you just need to call it once in the end of `NewModelV1beta1()` or `NewModelV1alpha1()`.

Reply:
Added a TODO. Feel free to remove the hold tag once all tests have passed. Thanks!

Reply:
Thanks!